app/src/main/java/org/mozilla/fenix/library/recentlyclosed/RecentlyClosedFragment.kt

```diff
@@ -41,6 +43,8 @@ class RecentlyClosedFragment : LibraryPageFragment<RecoverableTab>(), UserIntera
     private lateinit var recentlyClosedInteractor: RecentlyClosedFragmentInteractor
     private lateinit var recentlyClosedController: RecentlyClosedController
 
+    private lateinit var metrics: MetricController
```

Comment:
Do you still need this?  Looks like you can just use `requireComponents.analytics.metrics`

Reply:
There are 3 places where this is used in the fragment (two places where we track events, one where we pass the instance). Seems cleaner to use a use a property instead of accessors chains in all places.

Reply:
Sure.  Just thought that we can save an assignment with directly using it.  I'm OK with either.